lib/beaker/command.rb

```diff
@@ -148,9 +148,17 @@ def environment_string_for host, env
         env_array << "#{key.to_s.upcase}=\"#{val}\""
       end
 
-      environment_string = env_array.join(' ')
-
-      "env #{environment_string}"
+      if host['communicator'] =~ /bitvise/
```

Comment:
I still don't like the name 'communicator'.  We are looking to eliminate our cygwin dependency, so we would want to have PSWindows be the default, and old Windows be a fall through on failure.  Is there a way to do discoverability here?  Run a quick command to see if PSWindows connects and fall through otherwise?